_posts/2015-01-01-intro.md

```diff
@@ -22,23 +22,37 @@ style: left
 co-located with [FOSDEM 2016](https://fosdem.org/2016/). FOSDEM is an annual
 conference about free and open source software, attended by over 5000
 developers and open-source enthusiasts from all over the world. FOSDEM has
 hosted an [HPC devroom
 before](https://archive.fosdem.org/2014/schedule/track/hpc_and_computational_science),
-but this is the first time that HPC and Big Data open-source enthusiasts are
-joining forces to bring the two communities together with this devroom.
-
-- HPC and Big Data are two important approaches to scientific computing. HPC typically deals with smaller, higly structured data sets and huge amounts of computation while Big Data, not surprisingly, deals with gigantic, unstructured data sets and focuses on the IO bottlenecks. With Big Data trend unlocking access to an unprecedented amount of data, Data Science has emerged to tackle the problem of creating processes and approaches to extracting knowledge or insights from these data sets. Machine learning and predictive analytics are now the predominant type of computation performed on large clusters. We welcome talks about Free and Open Source solutions to the challenges presented by of large scale computing, data management and data analysis.
-
-- Free and Open Source communities have been the foundation of the HPC and Big Data communities for some time. In the HPC community, it's no surprise that [488 of the Top500 supercomputers run Linux](http://www.top500.org/statistics/details/osfam/1). On the Big Data side, the Hadoop ecosystem has had a tremendous amount of Open Source contributions from a wide range of organizations coming together under the Apache Software Foundation.
-
-- Our goal is to bring the communities together, share expertise, learn how we can benefit from each other's work and foster further joint research and collaboration.
-
-The devroom will take place on Sunday, January 31, at [ULB (Campus Solbosch)](https://www.openstreetmap.org/node/1632534522), in Brussels, Belgium. Join us to enjoy a full day of talks, demos
-and interesting discussions on open-source HPC, Big Data and Data Science.
+but this is the first time that representatives from the HPC and Big Data communities
+are joining forces to bring both communities together with this devroom.
+
+- High Performance Computing (HPC) and Big Data are two important approaches to scientific computing.
+  HPC typically deals with smaller, highly structured data sets and huge amounts of computation while
+  Big Data, not surprisingly, deals with gigantic, unstructured data sets and focuses on the I/O
+  bottlenecks. With the Big Data trend unlocking access to an unprecedented amount of data, Data
+  Science has emerged to tackle the problem of creating processes and approaches to extracting
+  knowledge or insights from these data sets. Machine learning and predictive analytics are now the
+  predominant type of computation performed on large clusters.
```

Comment:
Good point! How about: "Machine learning and predictive analytics algorithms have now joined the family of more traditional HPC algorithms and are pushing the requirements on cluster and data scalability"